optimum/intel/openvino/modeling_seq2seq.py

```diff
@@ -202,31 +203,32 @@ def __init__(
         self.decoder_with_past = None
         enable_compilation = kwargs.get("compile", True)
         encoder_cache_dir = Path(self.model_save_dir).joinpath("encoder_cache")
-        encoder_cache_dir.mkdir(parents=True, exist_ok=True)
-        ov_encoder_config = (
-            {**self.ov_config}
-            if "CACHE_DIR" in self.ov_config.keys()
-            else {**self.ov_config, "CACHE_DIR": str(encoder_cache_dir)}
-        )
+        ov_encoder_config = self.ov_config
```

Comment:
we don't want to modify `self.ov_config` by adding a cache path specific to the encoder / decoder component so I think we should create a copy that we can modify instead

Reply:
Thanks, I will fix that!